hwcomposer/hwcomposer_backend_v20.cpp

```diff
@@ -251,7 +251,7 @@ HwComposerBackend_v20::HwComposerBackend_v20(hw_module_t *hwc_module, void *libm
 
 HwComposerBackend_v20::~HwComposerBackend_v20()
 {
-    //hwc_device->eventControl(hwc_device, 0, HWC_EVENT_VSYNC, 0);
+    hwc2_compat_display_set_vsync_enabled(hwc2_primary_display, HWC2_VSYNC_DISABLE);
```

Comment:
This may or may not be necessary. I experienced a crash the very first time I restarted with just the change to stop minisf and so tried uncommenting this line too. But when I restarted to deploy this change I didn't see a crash so the first time around might just have been the result of some stale state. This still seems appropriate though, and I haven't been able to reproduce a crash in a dozen or so restarts with it.

Reply:
Scratch that, this is in fact the change that fixed the crash. My build of droidmedia that added the stopMiniSurfaceFlinger symbol was being built against an older sha1 so I was testing against a version that without the symbol.

Reply:
I think if this is the only change needed then we should drop the rest if possible. We've got a few combinations of surfaceflinger out there - minisf, libminisf and full sf, so stopping it could need testing on a few different devices, including community ones.

Reply:
I could be convinced either way. I've dropped the unloading since I'm not even sure the service shutdown() function does anything anyway (the base interface type has a static inline function which does nothing, but I presume the derivate types would overload that). 

Unloading the .so while code it provides is potentially still executing makes me nervous as does the general absence of tear down for the composer on destruction. But based on current evidence if those are real issues they're not frequently experienced.